docs/content/dagster-cloud/deployment/agents/customizing-configuration.mdx

```diff
@@ -94,3 +94,43 @@ compute_logs:
   module: dagster.core.storage.noop_compute_log_manager
   class: NoOpComputeLogManager
 ```
+
+---
+
+## Streaming compute logs to AWS S3
+
+You can set up streaming compute logs to AWS S3 by configuring the [dagster_aws.s3.compute_log_manager](https://docs.dagster.io/\_apidocs/libraries/dagster-aws#dagster_aws.s3.S3ComputeLogManager) module. Note that by setting the `show_url_only` parameter to `true`, only the URL of the log file will be shown in the Dagster Cloud UI instead of fetching and displaying the full content.
```

Comment:
We don't have the ability to fetch and display the full content since we don't have the s3 credentials - so i think its less that this is an optional thing that you can set, as that it is a required thing in order for there to be anything at all shown in compute logs